src/sql/workbench/parts/connection/common/connectionProviderExtension.ts

```diff
@@ -123,6 +124,7 @@ ExtensionsRegistry.registerExtensionPoint<ConnectionProviderProperties | Connect
 
 	for (let extension of extensions) {
 		const { value } = extension;
+		resolveIconPath(extension);
```

Comment:
Please update the `ConnectionProviderContrib` JSON schema on line 58 to include your new Icon JSON code. This enables extenders to open the file in ADS and understand what they can add. Also reach out to @anthonydresser  and @alanrenmsft to see if we need to update documentation for this

Reply:
I could not find `ConnectionProviderContrib.json` file. Where is it located?

Reply:
It's not a JSON file - it's the JSON schema definition on line 58 of this file. It is used to provide intellisense when you open this package.json in Azure Data Studio

Reply:
fixed.